Snippets/MetricsServiceControl/Metrics_1/Configuration.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Metrics_1
+{
+    using System;
+    using NServiceBus;
+
+    class Configuration
+    {
+        void ConfigureEndpoint(EndpointConfiguration endpointConfiguration)
+        {
+            #region SendMetricDataToServiceControl
+
+            var metrics = endpointConfiguration.EnableMetrics();
+#pragma warning disable 618
+            metrics.SendMetricDataToServiceControl(
+                serviceControlMetricsAddress: "address",
```

Comment:
Should we use the default queue value like in https://github.com/Particular/ServiceControl.Monitoring/pull/62